1039-wizard-design-guidance.md

```diff
@@ -0,0 +1,31 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-22
+---
+
+# Wizard Design Guidance
+
+The [wizard](https://clarity.design/documentation/wizard) component guides the user through a set of predefined steps. Wizards can help users focus on only the input needed for each step and decrease the chance of errors.
+
+## Guidance
+
+### Wizard Usage
+
+- **Must** have at least two steps in your wizard. 
+- **Must not** have long or complex content in each step of the wizard.
+- **Must not** use a wizard with an additional modal as context will be lost. It also blocks accessibility for keyboard users.
+- **Must not** use a wizard when the user needs to leave mid-workflow and resume later.
```

Comment:
```suggestion
- Design teams **must not** use a wizard when users need to leave mid-workflow and resume later, as this can disrupt their progress and lead to confusion. Instead, consider using a different structure that allows for saving progress.
```